<?xml version="1.0" encoding="UTF-8" ?>
<launch>
    <arg name="veh" default="husky1" />
    <arg name="port" default="/dev/glora" />
    <arg name="log" default="True" />

    <group ns="$(arg veh)">
        <node name="glora_client_node" pkg="lora_communication" type="glora_client.py" output="screen" required="true">
            <param name="veh" value="$(arg veh)" />
            <param name="port" value="$(arg port)" />
            <param name="log" value="$(arg log)" />
        </node>

        <!-- <node name="estop_control" pkg="control" type="estop_control.py" output="screen" /> -->
    </group>


</launch>